tests/functional/meson.build

```diff
@@ -115,7 +115,7 @@ suites = [
       'eval.sh',
       'short-path-literals.sh',
       'no-url-literals.sh',
-      'repl.sh',
+      # 'repl.sh',  # Temporarily disabled - failing test
```

Comment:
Idem.

Reply:
Those two fail locally for me still. I will drop the commit that comments them out before merging.